docs/getting-started/build-your-own-development-automation.md

```diff
@@ -217,6 +225,175 @@ Click on `Close` (you could also close the issue in GitHub if you prefer) and se
 
 There, you did it! You just created a new automation, and taught the Bot to listen for events and run that automation!
 
+## Convert an Existing Project of Your Own to be a new Atomist `generator`
+
+Now it's time to create your own project generator. Atomist generators take an existing, working project and convert them into the baseline for further projects, optionally editing parts of the baseline project where customisation is needed.
+
+!!! note ""
+    Unlike other project creation approaches, such as templating, Atomist generators don't break your project as they only add files in the `.atomist` directory and leave your project's code as-is. For this reason _any working project can be an Atomist generator_.
+
+For the simplest case to create a new generator of our own we will need to:
+
+- Take an existing project and convert it to being a Rug archive with a default generator that simply copies over the baseline project's files into a new project.
+- Publish your new generator so that it can be invoked from your team only.
+
+### Editing an Existing Project to make it a Generator
+
+First make sure you have already [installed the latest Rug CLI][cli-install]
+for your platform. You'll need the [Rug CLI][rug-cli] in order to publish your new rug archive.
+
+[rug-cli]: /user-guide/interfaces/cli/index.md
+[cli-install]: /user-guide/interfaces/cli/install.md
+
+Also make sure you already have your baseline project set to be converted into being a project generator. This means you have the baseline project already in GitHub in its own repository and that repository is connected to a channel in Slack that Atomist has been invited to.
+
+!!! note ""
+    We'll be converting an existing .NET project to becoming a generator but _Any_ existing project can be converted using this approach.
```

Comment:
I thought we had agreed to focus on Java? Not that I'm against .NET at all but I'd rather have consistency in our doc. That being said, this may not need to apply in the getting started, only in the user guides.